client/src/glossary/TooltipActivator.js

```diff
@@ -183,15 +104,26 @@ const TooltipActivator = _.isUndefined(MutationObserver)
           });
 
           outgoing_tooltips.forEach((outgoing_instance) =>
-            outgoing_instance.tooltip.dispose()
+            outgoing_instance.tooltip.destroy()
           );
 
           const incoming_tooltips = _.chain(current_tooltip_nodes)
             .without(..._.map(remaining_tooltips, (tooltip) => tooltip.node))
             .map((node) => ({
               node,
               glossary_key: node.getAttribute("data-ibtt-glossary-key"),
-              tooltip: new Tooltip(node, tt_params_from_node(node)),
+              tooltip: tippy(node, {
+                content: node.getAttribute("data-ibtt-glossary-key")
+                  ? get_glossary_item_tooltip_html(
+                      node.getAttribute("data-ibtt-glossary-key")
+                    )
+                  : node.getAttribute("data-ibtt-text"),
+                interactive: true,
+                duration: 0,
+                allowHTML: true,
+                hideOnClick: false,
+                appendTo: body, //silence the warning
```

Comment:
Repeat of lines 70 to 81, right? Would probably be good to DRY these, just to avoid them going out of sync when future developers make edits.

Reply:
Ah, haha, wait. Unless I'm missing something I think they already went out-of-sync in 
b32d3a6. Or are they actually supposed to be different configurations:?